### PR TITLE
Create the-journal-of-veterinary-medical-science.csl

### PR DESCRIPTION
**Pull request by shojitakahashi**
The Journal of Veterinary Medical Science
Publisher: Japanese Society of Veterinary Science
Journal homepage: http://jsvetsci.jp/jvms/
Instructions for Authors: http://jsvetsci.jp/jvms/instructions-for-authors/

### CODE REVIEW

File: the-journal-of-veterinary-medical-science.csl [hunk 1]
```diff
@@ -13,8 +13,8 @@
     </author>
     <category citation-format="numeric"/>
     <category field="medicine"/>
-    <issn>0021-4922</issn>
-    <eissn>1347-4065</eissn>
+    <issn>0916-7250</issn>
+    <eissn>1347-7439</eissn>
```
Comment (rmzelle) [inline the-journal-of-veterinary-medical-science.csl]: @shojitakahashi, the original ISSNs appear to be from the "Japanese Journal of Applied Physics" (compare http://iopscience.iop.org/journal/1347-4065 and https://www.jstage.jst.go.jp/browse/jvms).

**Comment by rmzelle** [comment]
> Is there anything else I need to do before this style is officially approved?

I just accepted the style. Thanks!

**Comment by shojitakahashi** [comment]
Thank you very much for correcting the ISSNs.  I will try to be careful next time.

**Comment by rmzelle** [comment]
No problem. Thanks for submitting.

**Pull request merged**